storageprovider/src/main/java/org/duracloud/storage/domain/StorageAccount.java

```diff
@@ -38,7 +38,30 @@
         BRIDGE_PORT,
         BRIDGE_USER,
         BRIDGE_PASS,
-        BRIDGE_MEMBER_ID;
+        BRIDGE_MEMBER_ID,
+        //GENERAL
+        WRITABLE_BY_NON_ROOT(false);
```

Comment:
Given that this parameter will be visible to all account users (via the REST API) when enabled, I'm wondering if the name should just be "WRITABLE"? That's not quite as descriptive for us, but might be more easily understandable.

Reply:
@bbranan :  that seems reasonable to me.